LICENSE

```diff
@@ -1,19 +1,21 @@
-Copyright (c) 2015, Brookhaven Science Associates, Brookhaven National
-Laboratory. All rights reserved.
+BSD 3-Clause License
+
+Copyright (c) 2015, Brookhaven National Lab
```

Comment:
Should it still have "Brookhaven Science Associates"?

Reply:
In [this issue](https://github.com/NSLS-II/scientific-python-cookiecutter/issues/89) @stuartcampbell said that "It should be Brookhaven National Laboratory" so I think it should be tweaked to the full word probably on rereading it.

Reply:
In my last conversation they were happy with simply "Brookhaven National Laboratory"

Reply:
Fixed to remove abbreviated form